doc/configuration.rst

```diff
@@ -1024,6 +1024,15 @@ Add additional HTTP headers to all requests to your sources.
 
 Sets the ``Access-control-allow-origin`` header to HTTP responses for `Cross-origin resource sharing <http://en.wikipedia.org/wiki/Cross-origin_resource_sharing>`_. This header is required for WebGL or Canvas web clients. Defaults to `*`. Leave empty to disable the header. This option is only available in `globals`.
 
+``manage_cookies``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
.rst requires that the ^^^ line length matches the title.

Reply:
Good catch - fixed